File: 251002/Troitsky/Lab1-5/Lab3.Core/Models/Post.cs
namespace Lab3.Core.Models
{
    public class Post
    {
        public ulong NewsId { get; } = 0;
        public string Content { get; } = string.Empty;
        private Post(ulong newsId, string content) => (NewsId, Content) = (newsId, content);
        static public Post Construct(ulong newsId, string content) 
        {
            return new Post(newsId, content);
        }
    }
}

File: 251002/Matalyga/Redis5/discussion/src/main/java/org/example/entities/Post.java
package org.example.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.cassandra.core.mapping.PrimaryKeyColumn;
import org.springframework.data.cassandra.core.mapping.Table;
import org.springframework.data.redis.core.RedisHash;

import java.io.Serializable;

import static org.springframework.data.cassandra.core.cql.PrimaryKeyType.CLUSTERED;
import static org.springframework.data.cassandra.core.cql.PrimaryKeyType.PARTITIONED;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(value = "tbl_post")
@RedisHash("post")
public class Post implements Serializable {

    @PrimaryKeyColumn(name = "country", type = PARTITIONED)
    private String country;

    @PrimaryKeyColumn(name = "newsid", ordinal = 0, type = CLUSTERED)
    private Long newsId;

    @PrimaryKeyColumn(name = "id", ordinal = 1, type = CLUSTERED)
    private Long id;

    private String content;
}